utils/add-chapters.js

```diff
@@ -0,0 +1,32 @@
+module.exports = async function addChapter(Module, Chapter) {
```

Comment:
Why is addChapter utils func needed?

Reply:
To seed the database with a list of chapters and modules that we have.

Reply:
Got it.